### DISCUSSION TITLE
Welcome to OPPS-Frontend Discussions!

### DISCUSSION THREAD

**Discussion by Muchael123** [Binary-Brigades/OPPS-Frontend#7] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Muchael123** [comment]
Nice

**Reply by Muchael123** [reply]
This is great

**Reply by David-mwas** [reply]
great

**Comment by David-mwas** [comment]
we're getting there🚀

**Reply by David-mwas** [reply]
nice
